demisto_sdk/commands/common/git_util.py

```diff
@@ -484,3 +484,25 @@ def _check_file_status(self, file_path: str, remote: str, branch: str) -> str:
             return ''
 
         return diff_line.split()[0].upper() if not diff_line.startswith('?') else 'A'
+
+    def get_local_remote_file_content(self, full_file_path: str, tag: str) -> Tuple[str, str]:
+        """Get local file content from remote branch. For example get origin/master:README.md
+
+        Args:
+            full_file_path: The file path to fetch. For example 'content/README.md'
+            tag: The tag of the branch. For example 'master'
+
+        Returns:
+            git_file_path: The git file path that was fetched. For example get origin/master:README.md
+            file_content: The fetched file content.
+        """
+        relative_file_path = full_file_path.split(f"{self.git_path()[-1]}/", 2)[-1]
```

Comment:
I think that it is much clearer and better to use `os.path.relpath` to the relative path of the file.

Reply:
You can also use Path(full_file_path).relative_to(self.git_path())